components/PhoneTrackingButton.vue

```diff
@@ -22,11 +25,25 @@ export default Vue.extend({
         // Remove the line before when you're implementing this
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
         this.watcherId = navigator.geolocation.watchPosition((result) => { /* Write your function here */ });
+        this.$bvToast.toast("You are being tracked.", {
+            title: "Paradar message",
+            autoHideDelay: 3000,
```

Comment:
The delay could be a bit longer for the 'you are being tracked'. For now, it is good definitely good enough though!